problemsets-mat223/modules/module6-exercises.tex

```diff
@@ -1,17 +1,86 @@
 \begin{exercises}
+	% Topics
+	% formal + intuitive definition of subspace
+	% relationship between subspace and spans
+	% prove a set is a subspace
+	% basis + dimension of a subspace
 	\begin{problist}
-		\prob  Example problem with parts
+
+		\prob If the following is a subspace, prove it. Otherwise, explain why not.
+		\begin{enumerate}
+			\item Let $\mathcal T \subseteq \R^2$ be the complete solution to $3x-y=0$.
+			\item Let $\mathcal U \subseteq \R^2$ be the complete solution to $\frac{1}{2}x-6y=0$.
+			\item Let $\mathcal V \subseteq \R^2$ be the complete solution to $x-5y-1=0$.
+			\item Let $\mathcal W \subseteq \R^3$ be the complete solution to $x+2y-3z=0$.
+			\item Let $\mathcal X \subseteq \R^3$ be the complete solution to
+			$5x-\pi y + (\ln 2)z=0$.
+			\item Let $n \in \N$. Let $a_1,...,a_n \in \R$. Let $\mathcal Q \subseteq \R^n$ be the complete solution to $a_1x_1+a_2x_2+...+a_{n-1}x_{n-1}+a_nx_n=0$.
+		\end{enumerate}
+
+		\prob For each of the following sets, prove whether or not it is a subpace.
+		\begin{enumerate}
+			\item Let $\mathcal B \subseteq \R^2$ be the the line in vector form
+			$\vec x = t\mat{5\\-7}+\mat{1\\2}$
+			\item Let $\mathcal A \subseteq \R^2$ be the the line in vector form
+			$\vec x = t\mat{-3\\4}$
+			\item Let $\mathcal C \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{1\\0\\5}$
+			\item Let $\mathcal D \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{2\\3\\4}+s\mat{10\\20\\131}+\mat{0\\0\\6}$
+			\item Let $\mathcal E \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{5\\7\\1}+s\mat{2\\-2\\1}$
+		\end{enumerate}
+
+		\prob Use the definition of subspace to show each span below is a subspace.
+		\begin{enumerate}
+			\item $\Span\Set*{\mat{0\\1}, \mat{1\\2}}$
+			\item $\Span\Set*{\mat{1\\1\\1}, \mat{1\\0\\0}, \mat{2\\0\\0}}$
+		\end{enumerate}
+
+		\prob For each set (a) find the dimension and (b) find two bases.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item $\Span\Set*{\mat{2\\3}, \mat{-4\\-6}, \mat{1\\\frac{3}{2}}}$
+			\item $\Span\Set*{\mat{1\\0\\-2}, \mat{0\\2\\5}, \mat{1\\2\\3}}$
+			\item $\vec x = t\mat{6\\1\\1}+s\mat{6\\0\\6}$
+			\item $\vec x = t\mat{2\\2\\3}$
+			\item $\mat{1\\-2\\3} \cdot \left(\mat{x\\y\\z}-\mat{2\\2\\\frac{2}{3}}\right)=0$
+			\item $\mat{1\\3\\3\\7} \cdot \left(\mat{x\\y\\z}-\mat{-7\\0\\0\\1}\right)=0$
 		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
 
+		\prob Which of the following are bases for $\R^3$?
```

Comment:
Did you check the numbers? Are they easy? They should start out easy, and you should have only one that involves a messy computation.